Modix.Data/Utilities/EntityExtensions.cs

```diff
@@ -24,8 +24,6 @@ public static double GetApprovalPercentage(this PromotionCampaignSummary campaig
                 .Where(x => x.Key == PromotionSentiment.Approve)
                 .Sum(x => x.Value);
 
-            if (totalApprovals < 1) return 0;
```

Comment:
What's the benefit of taking this out? iirc, we put this check in so we don't get a divide by zero exception when you have a campaign where the only vote is from the campaign creator, and they abstained